### DISCUSSION TITLE
[Bug Report][3.4.9] VSelect – using the value-prop, the v-select does not show the title of the selected value properly

### DISCUSSION THREAD

**Discussion by feam-codes** [vuetifyjs/vuetify#18986] [Q&A]
### Environment
**Vuetify Version:** 3.4.9
**Vue Version:** 3.4.5
**Browsers:** Chrome 120.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Add a VSelect with some items and define the initial value with the value-prop.

### Expected Behavior
The VSelect should show the title of the selected item.

### Actual Behavior
The VSelect shows the given value instead of the title.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNrFUsFOhDAQ/ZVJL2iyCwf1IGFNPOgXeFv20LCDNimlaQc0IXyPH+KPOYUl4LLxZjwNffOm770y+054VySP1sZtgyIVGWFltSR8yA1A1m6ltcPncChqQ1IZdCdoAD1qLGgCANptVR9R73JBThrPl6na5GImpIpFPPel1vU7Hl9mml/yAm3bSt0gc4e66pIiHbqEHzQ3k9/s/bk6S0xjFx/gv91NU6Vbe8qS858ckHEHsmSxG3z0hVOWwCM144qoytaOoAOHJfRQurqCiNcqCnQAvtcTLJ4EdoF5FZUuup4JF4IzcT/66ybDQ4gUIjTRZsJCUIaezKtW/m3C+1NdjbLs+egzi359SuV/Dh9C+DEuBxX9RtzEt/GdGOu9OHwDk60CaA==)

**Comment by KaelWD** [comment]
modelValue not value
